lib/DockerWrapper.js

```diff
@@ -61,6 +61,24 @@ const getCurrentServiceConfig = (service) => {
   });
 };
 
+const generateServiceEnvVars = async (app) => {
+    const hasDatabase = await app.getDatabase();
+    let envVariables = [];
+
+    if (hasDatabase) {
+      envVariables = [`DATABASE_HOST=${app.title}_database`,
+                      "POSTGRES_USER=postgres",
+                      "POSTGRES_PASSWORD=password",
+                      `POSTGRES_DB=${app.title}`];
+    }
+
+    if (app.envVariables) {
+      envVariables = [...envVariables, ...app.envVariables];
```

Comment:
We could also save the envVariables to the app at this point, then the database env variables would be stored in the database and rendered on the frontend (I think?). We could have a warning text on the frontend that says "don't change database environment variables!" Might be nice if they needed to look up the db connection details. Not sure on this one.

Reply:
Yeah that's a good point. I think in that case we could just save them to the app itself during the `createDatabase` process. If we _do_ go that route it invalidates the need for this function entirely.

This function was only created because we weren't already storing the DB environment variables on the app itself (thus, this function checks out the app and determines **all** the environment variables it needs, based on whether it has a DB and whether it has any custom env vars). If we're storing all environment variables, including those for the database, on the app itself we can just reference `app.envVariables` from the `updateService` method.